tket/src/Transformations/Decomposition.cpp

```diff
@@ -420,23 +420,6 @@ Transform decompose_tk1_to_rzrx() {
   });
 }
 
-Transform decompose_CX_to_ECR() {
```

Comment:
Could it be worth keeping this still?

Reply:
It's not used anwhere and we can easily restore if needed, so I think it's fine to go.